modules/configuration-api/src/main/java/org/apache/ignite/configuration/validation/ConfigurationValidationException.java

```diff
@@ -54,4 +55,9 @@ public ConfigurationValidationException(List<ValidationIssue> issues) {
     public List<ValidationIssue> getIssues() {
         return issues;
     }
+
+    private static String createMessageFromIssues(List<ValidationIssue> issues) {
```

Comment:
I don't think that this change is fully correct. It just skips error message from a validation issue.
In case of catching a such exception you need to add additional code to get all information. I mean that the following code will not be enough:
```
    try {
        ....
    }
    catch (Exception e) {
        log.("unexpected exception", e);
```
The code above will print: ```Validation did not pass for keys: Key 1, Key 2``. The problem itself will be lost.